src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.hpp

```diff
@@ -436,6 +437,9 @@ class TypeLookupManager
 
     //! Collection of all SampleIdentity and the TypeIdentfierWithSize it originated from, hashed by its SampleIdentity.
     std::unordered_map<SampleIdentity, xtypes::TypeIdentfierWithSize> async_get_type_requests_;
+
+    //! Max size of TypeLookup messages.
+    static constexpr uint32_t typelookup_data_max_size = 5000;
```

Comment:
do we really need it to be defined as static?

Reply:
If it isn't, then you cannot access it without a class instance, and since it is a constant and it came from a `#define` I think this is more suitable.